packages/core/nuxt-module/package.json

```diff
@@ -10,8 +10,8 @@
   "license": "MIT",
   "dependencies": {
     "@nuxt/typescript-build": "^2.0.0",
-    "@vue/composition-api": "1.0.0-beta.14",
-    "@nuxtjs/composition-api": "0.12.5",
+    "@vue/composition-api": "^1.0.0-beta.18",
+    "@nuxtjs/composition-api": "^0.13.5",
```

Comment:
@andrzejewsky afaik it shouldn't be hardcoded, right?

Reply:
@lukasborawski please remove both, our plugin is adding these ones